src/components/sharing/ShareButton/index.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+
+var _                       = require('lodash');
+var React                   = require('react/addons');
+var I18nMixin               = require('../../mixins/I18n');
+var Icon                    = require('../../helpers/Icon');
+var ShareBox                = require('../ShareBox');
+var format                  = require('../../../lib/format');
+var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
+
+var serviceConfigs = [
+  {
+    name: "facebook",
+    url: "http://www.facebook.com/sharer.php?u={url}"
+  },
+  {
+    name: "twitter",
+    url: "https://twitter.com/share?url={url}&text={title}"
+  },
+  {
+    name: "googleplus",
+    url: "https://plus.google.com/share?url={url}",
+    icon: "google-plus"
+  },
+  {
+    name: "pinterest",
+    url: "https://pinterest.com/pin/create/bookmarklet/?media={img}&url={url}&description={title}"
+  }
+];
+
+module.exports = React.createClass({
+  mixins: [I18nMixin],
+  displayName: "ShareButton",
+  propTypes: {
+    buttons: React.PropTypes.array,
+    shareUrl: React.PropTypes.string,
+    shareTitle: React.PropTypes.string,
+    shareImage: React.PropTypes.string,
+    renderIcon: React.PropTypes.bool,
+    i18n: React.PropTypes.object
+  },
+
+  getDefaultProps: function() {
+    return {
+      buttons: [
```

Comment:
perhaps `buttons: _.pluck(serviceConfigs, 'name'),` to avoid duplication and one less change point.